benchexec/tools/coveriteam.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.util as util
+
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+    Tool info for CoVeriTeam: On-Demand Composition of Cooperative Verification Systems.
+    URL: https://gitlab.com/sosy-lab/software/coveriteam.
+
+    This class has 2 purposes:
+        1. to serve as an abstract class for specific coveriteam programs like verifiers, validators, etc.
+        2. to serve as the tool info module for any generic coveriteam program.
```

Comment:
Is this still true now that `cmdline` has been removed? There is a default `cmdline` implementation, so it could be, I am just asking for confirmation.

Reply:
yes